tests/text/test_tabstops.py

```diff
@@ -34,9 +34,30 @@ def it_can_iterate_over_its_tab_stops(self, iter_fixture):
         assert count == expected_count
         assert TabStop_.call_args_list == expected_calls
 
+    def it_can_get_a_tab_stop_by_index(self, index_fixture):
+        tab_stops, expected_vals, tab_stop_, TabStop_, expected_calls = (
+            index_fixture
+        )
+        for count in range(len(tab_stops)):
+            assert tab_stops[count] is tab_stop_
+        assert TabStop_.call_args_list == expected_calls
+
     # fixture --------------------------------------------------------
 
     @pytest.fixture(params=[
+        ('w:pPr',                                              ()),
+        ('w:pPr/w:tabs/w:tab{w:pos=2880}',                     (2880,)),
+        ('w:pPr/w:tabs/(w:tab{w:pos=2880},w:tab{w:pos=5760})', (2880, 5760)),
+    ])
+    def index_fixture(self, request, TabStop_, tab_stop_):
+        pPr_cxml, expected_vals = request.param
+        pPr = element(pPr_cxml)
+        tab_elms = pPr.xpath('//w:tab')
+        tab_stops = TabStops(pPr)
+        expected_calls = [call(tab) for tab in tab_elms]
+        return tab_stops, expected_vals, tab_stop_, TabStop_, expected_calls
+
+    @pytest.fixture(params=[
```

Comment:
The fixture needs to be different, of course, because the items it supplies are different. But a couple remarks:

1. The values in the w:pos attributes are arbitrary. I'd use 1, 2, 3 ... so in case they show up in a test error you can identify the particular failing case easily. The return value is a tab_stop_ mock; tab_stop.position doesn't come into it yet.

2. The expected calls bit is relatively unusual. It shouldn't be needed here since there are no loops involved (if we don't put one in ourselves :). The .assert_called_once_with() method on the TabStops_ mock suffices up in the test itself.

Why don't you take another crack at this commit based on these comments and see what you can come up with.